<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_SNR</class>
 <widget class="QWidget" name="Form_SNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BOE SNR Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,2,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="8,2,1">
         <item>
          <widget class="QLineEdit" name="lineFolder"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectFolder">
           <property name="text">
            <string>Select Folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableCFG">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <column>
          <property name="text">
           <string>Config Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Config Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Customer</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>125</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>icons/BOE_logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Pattern</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cBoxDO">
            <property name="text">
             <string>Display Off</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cBoxWhite">
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="cBoxZ1">
            <property name="text">
             <string>Z1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cBoxZ4">
            <property name="text">
             <string>Z4</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="cBoxZ10">
            <property name="text">
             <string>Z10</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="cBoxZ2">
            <property name="text">
             <string>Z2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cBoxZ3">
            <property name="text">
             <string>Z3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="cBoxZ5">
            <property name="text">
             <string>Z5</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cBoxSlider">
            <property name="text">
             <string>Slider</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Additional Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QCheckBox" name="cBoxPlotallTouch">
        <property name="text">
         <string>Plot all Mct touch signal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cBoxPlotPosNoise">
        <property name="text">
         <string>Plot Mct positive/negative noise</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cBoxLog_grid_rawdata">
        <property name="text">
         <string>log grid rawdata</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cBoxPlotRms">
        <property name="text">
         <string>Plot nois_rms</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cBoxPlotNoiseptp">
        <property name="text">
         <string>Plot noise p2p</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cBoxPlot_p2p_annotated">
        <property name="text">
         <string>Plot noise p2p annotated</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cBoxPlotSctPosNoise">
        <property name="text">
         <string>Plot Sct positve/negative noise</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cBoxPlotSctSignal">
        <property name="text">
         <string>Plot all Sct touch signal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnReset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear Browser</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenerateSNR">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Generate SNR Report</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
